gosushi_client/src/components/Board.jsx

```diff
@@ -100,6 +109,7 @@ const Board = ({ socket }) => {
 
   return (
     <div className="board">
+      {!_.isEmpty(menu) && <Drawer menu={menu} />}
       <SpecialModal socket={socket} />
```

Comment:
I'm seeing an overflow issue on the menu
![image](https://user-images.githubusercontent.com/25088940/90985716-b4fcdd80-e54b-11ea-9fa0-db40c0fd05fb.png)
Adding an overflow-y: atuo, to modal body resolves the issue